Sources/AirbnbSwiftFormatTool/airbnb.swiftformat

```diff
@@ -25,6 +25,7 @@
 --typeattributes prev-line # wrapAttributes
 --wrapternary before-operators # wrap
 --structthreshold 20 # organizeDeclarations
+--sortswiftuiprops first-appearance-sort
```

Comment:
@calda I noticed there is a `#something` at the end of each, not sure what to add here.

Reply:
It's the name of the rule that uses the argument, so:

```suggestion
--sortswiftuiprops first-appearance-sort #organizeDeclarations
```

Reply:
I'd put the lower down below the `--typeorder` option